python/Ganga/Core/GangaRepository/GangaRepositoryXML.py

```diff
@@ -279,22 +281,22 @@ def index_load(self, this_id):
             logger.debug("Just silently continuing")
         return False
 
-    def index_write(self, this_id):
+    def index_write(self, this_id, shutdown=False):
```

Comment:
that it's run during shutdown as per convention elsewhere in the registry/repo where this has been used